### DISCUSSION TITLE
after transfer from original url, the branch and tag info of postcss-preset-env is missing

### DISCUSSION THREAD

**Discussion by LittleLittleBird12138** [csstools/postcss-plugins#1463] [General]
Why are all branches and tags lost after the postcss-preset-env code is migrated? Can you help restore it?
original url: https://github.com/csstools/postcss-preset-env
now url: https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env

**Comment by romainmenke** [comment]
@LittleLittleBird12138 Can you give some more information about what you are trying to do or which problem you are facing?

I am assuming that you aren't interested in the branches or tags themselves.
But rather that you need a specific tag or branch for some other reason?
